Userland/Utilities/less.cpp

```diff
@@ -288,8 +288,10 @@ class Pager {
         size_t n = 0;
         ssize_t size = getline(&line, &n, m_file);
 
-        if (size == -1)
+        if (size == -1) {
+            free(line);
```

Comment:
Please just use a `ScopeGuard` instead to add a single free that will be called at every exit of the function.